crates/astria-conductor/src/executor/mod.rs

```diff
@@ -67,6 +67,14 @@ pub(crate) struct StateNotInit;
 #[derive(Clone, Debug)]
 pub(crate) struct StateIsInit;
 
+#[derive(Debug, thiserror::Error)]
+pub(crate) enum StopHeightExceded {
```

Comment:
This shouldn't be used in conductor nor sequencer but remain private to this module.

Also, I don't think this should be an error as this is part of normal execution. As it currently stands, this will be emitted using an error event, which is not good.